airbyte-integrations/bases/connector-acceptance-test/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.8.3
+
+Add handling for global state messages.
```

Comment:
Wouldn't hurt to include your PR description here — call out that it's mostly used in DB sources.